tests/e2e/testfiles/scenarios.yaml

```diff
@@ -547,8 +551,10 @@
     - "Delete custom resource [1]"
     - "Restore template [testfiles/powerscale-templates/powerscale-secret-template.yaml] for [pscale]"
     - "Restore template [testfiles/powerscale-templates/karavi-authorization-config.json] for [pscaleAuthSidecar]"
+    - "Restore template [testfiles/powerscale-templates/powerscale-auth-secret-template.yaml] for [pscaleAuth]"
+    - "Restore template [testfiles/authorization-templates/rootCertificate.pem] for [authSidecarCert]"
 
-- scenario: "Install PowerScale Driver(With Authorization V1 and Observability), Disable Authorization module, Disable Observability module"
+- scenario: "Install PowerScale Driver(With Authorization V1 and Observability), Disable Observability module, Disable Authorization module"
```

Comment:
Do we document anywhere that observability must be removed before authorization? @falfaroc

Reply:
I believe that it would work but reconfiguration would be needed of the observability module which the tests currently don't support. To limit the changes, the order of disabling was changed.

We can either, leave this like this and update the tests in the future or invest more time into the tests for this release.

Reply:
Leaving it as-is seems fine for now.